src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -165,21 +175,25 @@ const VoiceFocusProvider: React.FC<Props> = ({ spec, options, children }) => {
   async function initVoiceFocus(
     vfSpec: VoiceFocusSpec | undefined,
     options: VoiceFocusDeviceOptions | undefined,
+    createMeetingResponse: any | undefined,
```

Comment:
Could we avoid `any` here?

Reply:
Keep same with JS SDK.
https://github.com/aws/amazon-chime-sdk-js/blob/147e7bb8f0b38e13410a50956c921d54cd872f44/src/voicefocus/VoiceFocusDeviceTransformer.ts#L138